theme/components/button.scss

```diff
@@ -120,6 +120,19 @@ a.ck-button {
 		}
 	}
 
+	&:hover {
+		@include ck-tooltip_visible();
+	}
+
+	// 1. Get rid of the focus outline around the tooltip when focused (but not :hover).
+	// 2. Don't display tooltips for disabled buttons, they're unavailable anyway.
```

Comment:
That's a good point. OTOH, I think tooltips should be hidden for disabled buttons, as a rule of thumb. 

There are 2 solutions to the problem:

1. Move the cases when the tooltips are disabled under `.ck-toolbar` and leave the `.ck-button` as generic as possible, with tooltips always enabled.
2. Enable tooltips using the mixin in your case.

WDYT?